Exersises/1.cpp

```diff
@@ -0,0 +1,26 @@
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
Usually 1.cpp contains implementations of functions described in 1.h. Executables should have their own names.

Reply:
It needs to use function freopen in Visual Studio